src/status_im/ui2/screens/chat/composer/input.cljs

```diff
@@ -172,21 +172,21 @@
                            :on-selection-change      (partial on-selection-change timeout-id last-text-change mentionable-users)
                            :on-change                (partial on-change last-text-change timeout-id mentionable-users refs chat-id sending-image)
                            :on-text-input            (partial on-text-input mentionable-users chat-id)}
-        children          (fn []
-                            (if mentions-enabled
-                              (for [[index [type text]] (map-indexed
-                                                         (fn [idx item]
-                                                           [idx item])
-                                                         (<sub [:chat/input-with-mentions]))]
-                                ^{:key (str index "_" type "_" text)}
-                                [rn/text (when (= type :mention) {:style {:color colors/primary-50}})
-                                 text])
-                              (get @input-texts chat-id)))]
+        children          (reagent/as-element
```

Comment:
just wondering what is the reason, because it worked fine before

Reply:
Simply removing `[]` from `children` fixes the issue but I wanted to convert it to a reagent element instead
https://github.com/status-im/status-mobile/commit/32d85d505986495190f05f7ea007c22dd48f88a3#r91057798